README.md

```diff
@@ -33,3 +33,18 @@ The Raspberry Pi hardware I2C does not support clock stretching. You have two op
 
 You may need to add your user to the i2c group, e.g. `sudo usermod -aG i2c nvidia`. It should just work after that.
 
+# ROS2
+## build with colcon
+`colcon build --event-handler console_direct+ --packages-select  imu_bno055`
+
+## launch
+`ros2 launch imu_bno055 imu.launch.py`
```

Comment:
maybe it's just me, but I'm philosophically opposed to extraneous dots in file names - call it imu_launch.py or IMUlaunch.py (but then caps... yuck)

Reply:
There's some disagreement in the community about this: https://discourse.ros.org/t/questioning-the-launch-py-file-extension/6352/8 .

Personally, I'm with you and prefer only a single separator, so I'd also prefer imu_launch.py

Reply:
I prefer imu_launch.py as well. I was under under the influence of robot_localization package where they use dot.

Reply:
No, I did not write code under influence.